db/migrate/20240809091311_backfill_assessment_comments.rb

```diff
@@ -0,0 +1,19 @@
+class BackfillAssessmentComments < ActiveRecord::Migration[7.1]
+  def up
```

Comment:
generally a good idea to [disable_ddl_transactions!](https://github.com/ankane/strong_migrations?tab=readme-ov-file#good-7)